guardrails/run/async_runner.py

```diff
@@ -244,7 +244,7 @@ async def async_step(
             raise e
         return iteration
 
-    @async_trace(name="call")
+    @async_trace(name="async_call")
```

Comment:
Rather than changing the name, can we set an attribute on the span instead indicating if it was sync or async?